packages/ag-charts-website/e2e/toolbar.spec.ts

```diff
@@ -1,29 +1,106 @@
 import { expect, test } from '@playwright/test';
 
-import { gotoExample, setupIntrinsicAssertions, toExamplePageUrls } from './util';
+import { gotoExample, setupIntrinsicAssertions, toExamplePageUrl } from './util';
 
 test.describe('toolbar', () => {
     setupIntrinsicAssertions();
 
-    const testUrls = toExamplePageUrls('financial-charts', 'financial-charts-showcase');
+    const { url } = toExamplePageUrl('financial-charts-test', 'e2e-toolbar', 'reactFunctionalTs');
```

Comment:
Nit: Could we use vanilla JS examples, I think this will scale better as there is less to load (no system JS involved)?

Reply:
Done